testsuite/tests/singlecluster/gateway/reconciliation/change_targetref/test_update_authpolicy_target_ref.py

```diff
@@ -4,11 +4,22 @@
 
 import pytest
 
+from testsuite.kuadrant.policy.authorization.auth_policy import AuthPolicy
+from .conftest import change_target_ref
+
 pytestmark = [pytest.mark.kuadrant_only, pytest.mark.dnspolicy]
 
 
+@pytest.fixture(scope="module")
+def authorization(oidc_provider, gateway, cluster, blame, module_label, route):  # pylint: disable=unused-argument
+    """Overwrite the authorization fixture and attach it to the gateway"""
+    policy = AuthPolicy.create_instance(cluster, blame("authz"), gateway, labels={"testRun": module_label})
+    policy.identity.add_oidc("default", oidc_provider.well_known["issuer"])
```

Comment:
Small tip: If you dont want to bother with `auth` fixture here, you can add "anonymous" identity which does not require auth :) But this is correct way too :+1: